public/electron.js

```diff
@@ -127,7 +127,9 @@ function createWindow() {
     minHeight: 700,
     titleBarStyle: process.platform === 'win32' ? 'default' : 'hidden',
     webPreferences: {
-      spellcheck: true
+      spellcheck: true,
+      nodeIntegration: true,
+      contextIsolation: false
```

Comment:
I read the docs of these 2 options, but I didn't rly get why they are needed. Could you give me a short explanation as to why you added them? Thank you!

https://www.electronjs.org/docs/latest/api/browser-window

Reply:
Ah, this can be removed - it was for deeplinking which I gutted because it was just too complicated to include with this PR. I'll remove it